docs/setup/settings.asciidoc

```diff
@@ -82,31 +82,48 @@ Elasticsearch nodes on startup.
-to specify a list of paths to the PEM file for the certificate authority for
-your Elasticsearch instance.
-
-`elasticsearch.ssl.keyPassphrase:`:: The passphrase that will be used to decrypt
-the private key. This value is optional as the key may not be encrypted.
-
-`elasticsearch.ssl.verificationMode:`:: *Default: full* Controls the
-verification of certificates presented by Elasticsearch. Valid values are `none`,
-`certificate`, and `full`. `full` performs hostname verification, and
-`certificate` does not.
+`elasticsearch.ssl.alwaysPresentCertificate:`:: *Default: false* Controls whether to always present the certificate specified by
+`elasticsearch.ssl.certificate` or `elasticsearch.ssl.keystore.path` when requested. This applies to all requests to Elasticsearch,
+including requests that are proxied for end-users. Setting this to `true` when Elasticsearch is using certificates to authenticate users can
+lead to proxied requests for end-users being executed as the identity tied to the configured certificate.
+
+`elasticsearch.ssl.certificate:` and `elasticsearch.ssl.key:`:: Paths to a PEM-encoded X.509 certificate and its private key, respectively.
+When `xpack.security.http.ssl.client_authentication` in Elasticsearch is set to `required`, the certificate and key are used to prove
+Kibana's identity when it makes an outbound request to your Elasticsearch cluster.
+
+NOTE: These settings cannot be used in conjunction with `elasticsearch.ssl.keystore.path`.
+
+`elasticsearch.ssl.certificateAuthorities:`:: Paths to one or more PEM-encoded X.509 certificates. This may consist of a root certificate
+authority (CA), and one or more intermediate CAs, which make up a trusted certificate chain for Kibana. This is used to establish trust when
+Kibana creates an SSL connection with your Elasticsearch cluster. In addition to this setting, trusted certificates may be specified via
+`elasticsearch.ssl.keystore.path` and/or `elasticsearch.ssl.truststore.path`.
+
+`elasticsearch.ssl.keyPassphrase:`:: The passphrase that will be used to decrypt the private key that is specified via
+`elasticsearch.ssl.key`. This value is optional, as the key may not be encrypted.
+
+`elasticsearch.ssl.keystore.path:`:: Path to a PKCS #12 file -- this should contain an X.509 certificate with its private key. When
+`xpack.security.http.ssl.client_authentication` in Elasticsearch is set to `required`, the certificate and key are used to prove Kibana's
+identity when it makes an outbound request to your Elasticsearch cluster. If the file contains any additional certificates, those will be
+used as a trusted certificate chain for your Elasticsearch cluster. This is used to establish trust when Kibana creates an SSL connection
+with your Elasticsearch cluster. In addition to this setting, trusted certificates may be specified via
+`elasticsearch.ssl.certificateAuthorities` and/or `elasticsearch.ssl.truststore.path`.
+
```

Comment:
To have this NOTE show up properly indented, you need to tie it to the previous paragarph:
```suggestion
+
--
```